README.rst

```diff
@@ -0,0 +1,88 @@
+azure_lightning_flask
+=====================
+
+A `Flask`_ server to serve single-page applications deployed using the
+`lightning strategy`_ on `Azure Table Storage`_. Designed to work with
+`Ember-cli-deploy-azure`_, but should be compatible with any lightning
+strategy deployment on Azure Table Storage via configuration. Developed
+with Python 2.7
+
+Installation
+------------
+
+Manual
+~~~~~~
+
+Download the application files and in the root directory run
+
+::
+
+    pip install -r requirements.txt
+    python setup.py install
+
```

Comment:
If you want to include venv instructions, which I recommend but it's entirely up to you:

```
$ virtualenv -p /usr/bin/python2.7 venv
$ . venv/bin/activate
(venv) $ pip install -r requirements.txt
(venv) $ python setup.py install
```
(you'd have to run the app and tests from venv as well)

Reply:
I'd add this.

Reply:
Added